### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by aliabbani**
## Bookstore: styling

In this milestone we:

- All the details of the Bookstore design in Zeplin.
- Used the information detailed in Zeplin to style the website.
- The final design should match Zeplin's design.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,3 +1,202 @@
...
+.list-left-category {
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+}
+
+.list-left-title {
+  font-size: 1.375rem;
+  font-family: Roboto Slab, Arial, sans-serif;
```
Comment (wandji20) [inline src/index.css]: - [ ] It will be nice to create a class for this property for easy re-use since you have multiple declarations for this style.

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @aliabbani  :raised_hands: 
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for this project.
However, there are a few issues you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Very descriptive commit messages and pull request summary

## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.
Optional comments are not required to mark this project approved
- [ ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .hintrc, .eslintrc.json etc).
- [ ] Our navbar and book progress status are a bit out of place. Please compare the snapshots and make the required changes. 
- Your project!
Screenshot from 2021-10-01 12-49-34
- Expected design
Screenshot from 2021-10-01 12-50-10

**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (aliabbani) [reply src/index.css]: Hello @wandji20 can you explain what you mean by ignoring all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .hintrc, .eslintrc.json etc). Do you mean that I have to disable linters ?

Comment (wandji20) [reply src/index.css]: Hi @aliabbani,
The files I mentioned are required to run Linters locally. These you can ignore, yet keep in mind that your GitFlow configuration is a project requirement.

Comment (aliabbani) [reply src/index.css]: Alright thank you

File: src/index.css [hunk 2]
```diff
@@ -1,3 +1,217 @@
...
+}
+
+.list-left-title,
+.list-left-author,
+.list-left-button,
+.list-left-button2,
+.list-right-comp,
+.list-right-comp1 {
+  font-family: Roboto Slab, Arial, sans-serif;
+}
```
Comment (helman101) [inline src/index.css]: - [x] I would recommend you to do as my colleague said, please create a class for this property. 🤓 
    - For example, a class called `font-roboto` and then use this class in each element that needs this font.

Comment (helman101) [inline src/index.css]: - [ ] [OPTIONAL] As an optional change I will recommend taking a look at all your CSS files and find that repeated properties and then change them for classes as i have explain in the comment above 😄

File: .gitignore [hunk 3]
```diff
@@ -21,3 +21,9 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+####
+.lintrc
+.stylelintrc.json
+.hintrc
+.eslintrc.json
```
Comment (helman101) [inline .gitignore]: - [ ] Even though you have put these files in the .gitignore I still see them in the Github repository, please remove them. 😸

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @aliabbani   👋,

Great work on making the changes requested by a previous reviewer 👏 👏👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions 🤓 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 2 above)
Comment (aliabbani) [reply src/index.css]: Thank you, I will do that

File: .gitignore (hunk 3 above)
Comment (aliabbani) [reply .gitignore]: Alright

**Review by barackm — APPROVED** [review]
Hi @aliabbani, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**